libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -36,47 +36,30 @@ $min-screen-width: 375px;
       border-color: utils.get-color('danger');
     }
   }
-}
-
-:host > button {
-  position: relative;
-  margin: 0;
-  outline: none;
-  width: 100%;
-
-  // Temporary fix for button-width as attention level 3 has border,
-  // and attention level 2 does not:
-  &.attention-level2 {
-    border: 1px solid transparent;
-  }
 
-  .text {
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-}
+  button {
```

Comment:
Strictly speaking, this selector should be `> button `, shouldn't it? In the "before refactor" it uses a [child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) ➡️ "buttons that are _direct_ children of :host"